cmake/modules/SwiftStripBuilderRpath.cmake

```diff
@@ -0,0 +1,71 @@
+set(SWIFT_SET_RPATH_SCRIPT_FILE "${CMAKE_CURRENT_LIST_FILE}")
+
+function(swift_get_set_rpath_script_file out_var)
+  set(${out_var} "${SWIFT_SET_RPATH_SCRIPT_FILE}" PARENT_SCOPE)
+endfunction()
+
+# Actual RPATH_SET operation to the file.
+function(_swift_set_rpath_impl file new_rpath)
+  # NOTE: RPATH_SET is not documented, and works only for ELF and XCOFF. 
+  file(RPATH_SET FILE "${file}" NEW_RPATH "${new_rpath}")
+endfunction()
+
+# For 'install(SCRIPT <script> CODE swift_set_rpath(...)'.
+function(_swift_file_set_rpath_installed file new_rpath)
+  set(DESTDIR $ENV{DESTDIR})
+  if(NOT IS_ABSOLUTE "${file}")
+    string(PREPEND file "${CMAKE_INSTALL_PREFIX}/")
+  endif()
+  string(PREPEND file "${DESTDIR}")
+  _swift_set_rpath_impl("${file}" "${new_rpath}")
+endfunction()
+
+# Add 'install' script that set runtime path to the specified file.
+function(swift_install_file_set_rpath file install_rpath component)
+  if(NOT(SWIFT_HOST_VARIANT_SDK STREQUAL "LINUX"))
+      return()
+  endif()
+  if((NOT SWIFT_SWIFT_PARSER) AND NOT(BOOTSTRAPPING_MODE STREQUAL "HOSTTOOLS"))
+      return()
+  endif()
```

Comment:
My intent was  `!(linux && (swift_swift_parser || boostrapping==hosttools))` because `boostrapping==hosttools` alone also needs to strip builder's `RUNPATH`.

Reply:
`bootstrapping==hosttools` without early swift syntax shouldn't have the host in the `RUNPATH` anyway though right?

Reply:
It should, otherwise which runtime does the `SwiftCompilerSources` library link with (before it builds stdlib/runtime)?

Reply:
Ah true, hosttools uses SwiftCompilerSources too 😅 Thanks, sorry for the run around!